generators/app/templates/resources/lib/utilities.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import xbmc
+import xbmcaddon
+import re
+import sys
+import logging
+
+
+if sys.version_info >= (2, 7):
+    import json as json
+else:
+    import simplejson as json
```

Comment:
Get rid of this as it's obsolete since v17 and we should not promote usage anymore